### DISCUSSION TITLE
How to update div id q-app

### DISCUSSION THREAD

**Discussion by prasadregula** [quasarframework/quasar#16974] [General - Components / Directives / etc]
I have to deploy 2 Quasar apps in the same page. But to ID conflict both the apps are overwriting. i need help to solve this issue.

**Comment by yusufkandemir** [comment]
See this example: https://github.com/quasarframework/quasar/blob/59948ed5628dd86383cab74ea625ac56827a6cd1/ui/dev-umd/index-multiapp.umd.html

Side note: it's not possible to change it with Quasar CLI as it produces a whole app with an index.html file and other assets. You are probably not using this approach, since you are trying to put two apps on the same page, but wanted to state it anyway.

**Reply by prasadregula** [reply]
Thanks for your update. if is possible to change on build time?

**Reply by yusufkandemir** [reply]
Not sure what you mean. I also don't have any information about your project, build system, etc. because you didn't share it. So, I can't help further than this.
